mk/Common.mk

```diff
@@ -184,11 +184,7 @@ endif
 ###############################################################################
 # Imperas Instruction Set Simulator
 
-DV_OVPM_HOME    = $(CORE_V_VERIF)/vendor_lib/imperas
-DV_OVPM_MODEL   = $(DV_OVPM_HOME)/imperas_DV_COREV
-DV_OVPM_DESIGN  = $(DV_OVPM_HOME)/design
-OVP_MODEL_DPI   = $(DV_OVPM_MODEL)/bin/Linux64/imperas_CV32.dpi.so
-#OVP_CTRL_FILE   = $(DV_OVPM_DESIGN)/riscv_CV32E40P.ic
+IMPERAS_DV_MODEL = $(IMPERAS_HOME)/lib/$(IMPERAS_ARCH)/ImperasLib/imperas.com/verification/riscv/1.0/model.so
```

Comment:
This file is used by all other cores that use CORE-V-VERIF.  Is this path the same as all others?  If we can't know that, then this should do in `mk/uvmt/uvmt.mk`.

Reply:
@duncangraham-Imperas you deleted some DV_OVPM variables, do you know that they are not being used by anyone else?

Reply:
These variables all relate to the old deprecated way in which an Imperas reference model could be instanced and used in a System Verilog simulation. Support for this method was removed some time ago. Anyone using and developing in the current repository should not be using this method. If they are using this method, I expect they would also be using a repository containing the version of the model corresponding to it.
I believe it would be better for them to have an error than to use an out of date reference model from the repo when developing new or making modifications to RTL.
Does that make sense?

Reply:
> I believe it would be better for them to have an error than to use an out of date reference model from the repo when developing new or making modifications to RTL.

Agreed.  Good suggestion @duncangraham-Imperas.